docs/Neogeo-CD.md

```diff
@@ -0,0 +1,46 @@
+***
+_ The Neo Geo CD is the second home video game console of SNK Corporation's Neo Geo family, released on September 9, 1994, four years after its cartridge-based equivalent. The system was originally priced at US$399, or £399. It was discontinued in 1997 with sales of 570,000 systems._
+
+***
+
+| Emulator | Rom Folder | Extension | BIOS |  Controller Config |
+| :---: | :---: | :---: | :---: | :---: |
+| [lr-neocd](https://github.com/libretro/neocd_libretro) | neogeo  | .chd .cue .zip | uni-bioscd.rom | /opt/retropie/configs/neogeo/retroarch.cfg |
+
+## Emulator: [lr-neocd](https://github.com/libretro/neocd_libretro) 
+
+**lr-neocd**  is a complete rewrite of NeoCD from scratch in modern C++11. It is designed with accuracy and portability in mind rather than being all about speed like the the older versions.
+
+## ROMS
+Accepted File Extensions: **.cue** **.chd** **.zip**
+
+Place your Neogeo CD ROMs in
+```
+/home/pi/RetroPie/roms/neogeo
+```
+## BIOS
+You will need a minimum of two BIOS files (eg. ng-lo.rom, uni-bioscd.rom) which should be copied to $biosdir/neocd.
```

Comment:
Why 2 BIOS files are needed ? The core documentation only mentions one. Also, as said in the general comments, expand `$biosdir`.

Reply:
I was quoting the installation script for neocd https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/libretrocores/lr-neocd.sh

Reply:
Sorry, but I don't follow. The ROMs folder is specified completely with the path, so is the BIOS folder in the other emulator/ports pages. Please do so for this page also.